src/components/layout/layout-v2.tsx

```diff
@@ -111,31 +81,6 @@ interface LayoutProps {
   breadcrumb?: BreadcrumbEntry[];
 }
 
-enum POLARITY {
-  DARK = 'dark',
-  LIGHT = 'light',
-}
-
-function setPolarityBodyClass(isLight: boolean) {
-  const POLARITY_PREFIX = `sy-polarity--`;
-  const classNameDark = `${POLARITY_PREFIX}${POLARITY.DARK}`;
-  const classNameLight = `${POLARITY_PREFIX}${POLARITY.LIGHT}`;
-
-  const noBrowser = typeof window === 'undefined';
-
-  if (noBrowser) {
-    return;
-  }
-
-  if (isLight) {
-    document.body.classList.remove(classNameDark);
-    document.body.classList.add(classNameLight);
-  } else {
-    document.body.classList.remove(classNameLight);
-    document.body.classList.add(classNameDark);
-  }
-}
-
```

Comment:
separate file because it's cluttering the view and deserves its own file for sure